api/controllers/FlairController.js

```diff
@@ -168,8 +168,8 @@ module.exports = {
         }
       });
 
-      var newPFlair = _.get(req, "user.flair.ptrades.current.flair_css_class") || "default";
-      var newsvFlair = _.get(req, "user.flair.svex.current.flair_css_class") || "";
+      var newPFlair = _.get(req, "user.flair.ptrades.flair_css_class", "default");
```

Comment:
Hmmmmm. I'm not convinced, because this is where I did it. But alright, I'll take your word for it.